src/main/java/com/github/ozsie/CheckMojo.kt

```diff
@@ -15,18 +16,29 @@ import java.nio.file.Paths
         threadSafe = true,
         requiresDependencyCollection = ResolutionScope.TEST)
 class CheckMojo : DetektMojo() {
+    lateinit var cliArgs: CliArgs
```

Comment:
set to a field for testing purposes. Maybe there is another smart idea of how we can assert the expected outcome.